stories/CyVerseAppBar.stories.js

```diff
@@ -1,11 +1,32 @@
 import React, { Component } from "react";
 import { storiesOf } from "@storybook/react";
 import CyverseAppBar from "../src/components/appBar/CyVerseAppBar";
+import { UserProfileProvider } from "../src/contexts/userProfile";
+import fetchMock from "fetch-mock";
+
+const mockUser = {
+    id: "mockUser",
```

Comment:
Your mother was a 🐹 and your father smelled of 🍓

Reply:
🤣

Reply:
Now I'm waiting to see when we pull the farm-animal catapults out. 😆